.gitignore

```diff
@@ -1,3 +1,15 @@
+# generated UIs
+# pytripgui/view/gen/field.py
+# pytripgui/view/gen/kernel.py
+# pytripgui/view/gen/main_window.py
+# pytripgui/view/gen/plan.py
+# pytripgui/view/gen/trip_config.py
+
+# versioneer stuff  # TODO find some solution
```

Comment:
versioneer cannot be exclude as it needs to add himself temporarily to versioneed files to figureout the version number.
The idea would be to get rid of versioneer at all

Reply:
Until then, a hack could be to add some of the generated files to the repo? The contents of those files does not seem to change anyway, so maybe it is acceptable.